2024-12-10 06:25:03 - INFO - Directory: py-cpp
2024-12-10 06:25:03 - INFO - Total Files: 49
2024-12-10 06:25:03 - INFO - Successful Compilations: 32
2024-12-10 06:25:03 - INFO - Failed Compilations: 17
2024-12-10 06:25:03 - INFO - Success Rate: 65.31%
2024-12-10 11:12:29 - ERROR - Failed to compile py-cpp/HumanEval_27_flip_case.cpp. Error:
py-cpp/HumanEval_27_flip_case.cpp: In function ‘std::string flip_case(std::string)’:
py-cpp/HumanEval_27_flip_case.cpp:112:1: error: expected initializer before ‘}’ token
  112 | }
      | ^
py-cpp/HumanEval_27_flip_case.cpp:112:1: warning: no return statement in function returning non-void [-Wreturn-type]

2024-12-10 11:12:29 - INFO - Successfully compiled: py-cpp/HumanEval_83_starts_one_ends.cpp
2024-12-10 11:12:30 - INFO - Successfully compiled: py-cpp/HumanEval_143_words_in_sentence.cpp
2024-12-10 11:12:30 - ERROR - Failed to compile py-cpp/HumanEval_51_remove_vowels.cpp. Error:
py-cpp/HumanEval_51_remove_vowels.cpp: In function ‘std::string remove_vowels(std::string)’:
py-cpp/HumanEval_51_remove_vowels.cpp:109:1: error: expected initializer before ‘}’ token
  109 | }
      | ^
py-cpp/HumanEval_51_remove_vowels.cpp:109:1: warning: no return statement in function returning non-void [-Wreturn-type]

2024-12-10 11:12:31 - INFO - Successfully compiled: py-cpp/HumanEval_135_can_arrange.cpp
2024-12-10 11:12:32 - ERROR - Failed to compile py-cpp/HumanEval_131_digits.cpp. Error:
py-cpp/HumanEval_131_digits.cpp: In function ‘long int digits(long int)’:
py-cpp/HumanEval_131_digits.cpp:159:10: error: expected ‘)’ before ‘}’ token
  159 |     if (n
      |        ~ ^
      |          )
  160 | }
      | ~         
py-cpp/HumanEval_131_digits.cpp:160:1: error: expected primary-expression before ‘}’ token
  160 | }
      | ^
py-cpp/HumanEval_131_digits.cpp:160:1: warning: control reaches end of non-void function [-Wreturn-type]
  160 | }
      | ^

2024-12-10 11:12:33 - ERROR - Failed to compile py-cpp/HumanEval_62_derivative.cpp. Error:
py-cpp/HumanEval_62_derivative.cpp:67:21: warning: missing terminating " character
   67 |     std::regex rx51("[0-
      |                     ^
py-cpp/HumanEval_62_derivative.cpp:67:21: error: missing terminating " character
   67 |     std::regex rx51("[0-
      |                     ^~~~
py-cpp/HumanEval_62_derivative.cpp: In function ‘std::vector<long int> derivative(std::vector<long int>)’:
py-cpp/HumanEval_62_derivative.cpp:68:1: error: expected primary-expression before ‘}’ token
   68 | }
      | ^
py-cpp/HumanEval_62_derivative.cpp:68:1: error: expected ‘,’ or ‘;’ before ‘}’ token
py-cpp/HumanEval_62_derivative.cpp:68:1: warning: no return statement in function returning non-void [-Wreturn-type]

2024-12-10 11:12:33 - INFO - Successfully compiled: py-cpp/HumanEval_6_parse_nested_parens.cpp
2024-12-10 11:12:34 - INFO - Successfully compiled: py-cpp/HumanEval_1_separate_paren_groups.cpp
2024-12-10 11:12:35 - INFO - Successfully compiled: py-cpp/HumanEval_42_incr_list.cpp
2024-12-10 11:12:35 - INFO - Successfully compiled: py-cpp/HumanEval_40_triples_sum_to_zero.cpp
2024-12-10 11:12:36 - ERROR - Failed to compile py-cpp/HumanEval_121_solution.cpp. Error:
py-cpp/HumanEval_121_solution.cpp: In function ‘long int solution(std::vector<long int>)’:
py-cpp/HumanEval_121_solution.cpp:21:15: error: ‘std::__cxx11::smatch’ {aka ‘class std::__cxx11::match_results<__gnu_cxx::__normal_iterator<const char*, std::__cxx11::basic_string<char> > >’} has no member named ‘first’
   21 |             m.first = std::stoi(m.second);
      |               ^~~~~
py-cpp/HumanEval_121_solution.cpp:21:35: error: ‘std::__cxx11::smatch’ {aka ‘class std::__cxx11::match_results<__gnu_cxx::__normal_iterator<const char*, std::__cxx11::basic_string<char> > >’} has no member named ‘second’
   21 |             m.first = std::stoi(m.second);
      |                                   ^~~~~~
py-cpp/HumanEval_121_solution.cpp:22:15: error: ‘std::__cxx11::smatch’ {aka ‘class std::__cxx11::match_results<__gnu_cxx::__normal_iterator<const char*, std::__cxx11::basic_string<char> > >’} has no member named ‘second’
   22 |             m.second = std::to_string(m.first);
      |               ^~~~~~
py-cpp/HumanEval_121_solution.cpp:22:41: error: ‘std::__cxx11::smatch’ {aka ‘class std::__cxx11::match_results<__gnu_cxx::__normal_iterator<const char*, std::__cxx11::basic_string<char> > >’} has no member named ‘first’
   22 |             m.second = std::to_string(m.first);
      |                                         ^~~~~
py-cpp/HumanEval_121_solution.cpp:24:20: error: ‘std::__cxx11::smatch’ {aka ‘class std::__cxx11::match_results<__gnu_cxx::__normal_iterator<const char*, std::__cxx11::basic_string<char> > >’} has no member named ‘first’
   24 |         s.insert(m.first);
      |                    ^~~~~
py-cpp/HumanEval_121_solution.cpp:25:11: error: ‘std::__cxx11::smatch’ {aka ‘class std::__cxx11::match_results<__gnu_cxx::__normal_iterator<const char*, std::__cxx11::basic_string<char> > >’} has no member named ‘first’
   25 |         m.first = 0;
      |           ^~~~~
py-cpp/HumanEval_121_solution.cpp:26:11: error: ‘std::__cxx11::smatch’ {aka ‘class std::__cxx11::match_results<__gnu_cxx::__normal_iterator<const char*, std::__cxx11::basic_string<char> > >’} has no member named ‘second’
   26 |         m.second = 0;
      |           ^~~~~~
py-cpp/HumanEval_121_solution.cpp:27:11: error: ‘std::__cxx11::smatch’ {aka ‘class std::__cxx11::match_results<__gnu_cxx::__normal_iterator<const char*, std::__cxx11::basic_string<char> > >’} has no member named ‘first’
   27 |         m.first = std::stoi(m.second);
      |           ^~~~~
py-cpp/HumanEval_121_solution.cpp:27:31: error: ‘std::__cxx11::smatch’ {aka ‘class std::__cxx11::match_results<__gnu_cxx::__normal_iterator<const char*, std::__cxx11::basic_string<char> > >’} has no member named ‘second’
   27 |         m.first = std::stoi(m.second);
      |                               ^~~~~~
py-cpp/HumanEval_121_solution.cpp:28:11: error: ‘std::__cxx11::smatch’ {aka ‘class std::__cxx11::match_results<__gnu_cxx::__normal_iterator<const char*, std::__cxx11::basic_string<char> > >’} has no member named ‘second’
   28 |         m.second = std::to_string(m.first);
      |           ^~~~~~
py-cpp/HumanEval_121_solution.cpp:28:37: error: ‘std::__cxx11::smatch’ {aka ‘class std::__cxx11::match_results<__gnu_cxx::__normal_iterator<const char*, std::__cxx11::basic_string<char> > >’} has no member named ‘first’
   28 |         m.second = std::to_string(m.first);
      |                                     ^~~~~
py-cpp/HumanEval_121_solution.cpp:29:11: error: ‘std::__cxx11::smatch’ {aka ‘class std::__cxx11::match_results<__gnu_cxx::__normal_iterator<const char*, std::__cxx11::basic_string<char> > >’} has no member named ‘first’
   29 |         m.first = 0;
      |           ^~~~~
py-cpp/HumanEval_121_solution.cpp:30:11: error: ‘std::__cxx11::smatch’ {aka ‘class std::__cxx11::match_results<__gnu_cxx::__normal_iterator<const char*, std::__cxx11::basic_string<char> > >’} has no member named ‘second’
   30 |         m.second = 0;
      |           ^~~~~~
py-cpp/HumanEval_121_solution.cpp:31:11: error: ‘std::__cxx11::smatch’ {aka ‘class std::__cxx11::match_results<__gnu_cxx::__normal_iterator<const char*, std::__cxx11::basic_string<char> > >’} has no member named ‘first’
   31 |         m.first = std::stoi(m.second);
      |           ^~~~~
py-cpp/HumanEval_121_solution.cpp:31:31: error: ‘std::__cxx11::smatch’ {aka ‘class std::__cxx11::match_results<__gnu_cxx::__normal_iterator<const char*, std::__cxx11::basic_string<char> > >’} has no member named ‘second’
   31 |         m.first = std::stoi(m.second);
      |                               ^~~~~~
py-cpp/HumanEval_121_solution.cpp:32:11: error: ‘std::__cxx11::smatch’ {aka ‘class std::__cxx11::match_results<__gnu_cxx::__normal_iterator<const char*, std::__cxx11::basic_string<char> > >’} has no member named ‘second’
   32 |         m.second = std::to_string(m.first);
      |           ^~~~~~
py-cpp/HumanEval_121_solution.cpp:32:37: error: ‘std::__cxx11::smatch’ {aka ‘class std::__cxx11::match_results<__gnu_cxx::__normal_iterator<const char*, std::__cxx11::basic_string<char> > >’} has no member named ‘first’
   32 |         m.second = std::to_string(m.first);
      |                                     ^~~~~
py-cpp/HumanEval_121_solution.cpp:33:11: error: ‘std::__cxx11::smatch’ {aka ‘class std::__cxx11::match_results<__gnu_cxx::__normal_iterator<const char*, std::__cxx11::basic_string<char> > >’} has no member named ‘first’
   33 |         m.first = 0;
      |           ^~~~~
py-cpp/HumanEval_121_solution.cpp:34:11: error: ‘std::__cxx11::smatch’ {aka ‘class std::__cxx11::match_results<__gnu_cxx::__normal_iterator<const char*, std::__cxx11::basic_string<char> > >’} has no member named ‘second’
   34 |         m.second = 0;
      |           ^~~~~~
py-cpp/HumanEval_121_solution.cpp:35:11: error: ‘std::__cxx11::smatch’ {aka ‘class std::__cxx11::match_results<__gnu_cxx::__normal_iterator<const char*, std::__cxx11::basic_string<char> > >’} has no member named ‘first’
   35 |         m.first = std::stoi(m.second);
      |           ^~~~~
py-cpp/HumanEval_121_solution.cpp:35:31: error: ‘std::__cxx11::smatch’ {aka ‘class std::__cxx11::match_results<__gnu_cxx::__normal_iterator<const char*, std::__cxx11::basic_string<char> > >’} has no member named ‘second’
   35 |         m.first = std::stoi(m.second);
      |                               ^~~~~~
py-cpp/HumanEval_121_solution.cpp:36:11: error: ‘std::__cxx11::smatch’ {aka ‘class std::__cxx11::match_results<__gnu_cxx::__normal_iterator<const char*, std::__cxx11::basic_string<char> > >’} has no member named ‘second’
   36 |         m.second = std::to_string(m.first);
      |           ^~~~~~
py-cpp/HumanEval_121_solution.cpp:36:37: error: ‘std::__cxx11::smatch’ {aka ‘class std::__cxx11::match_results<__gnu_cxx::__normal_iterator<const char*, std::__cxx11::basic_string<char> > >’} has no member named ‘first’
   36 |         m.second = std::to_string(m.first);
      |                                     ^~~~~
py-cpp/HumanEval_121_solution.cpp:37:11: error: ‘std::__cxx11::smatch’ {aka ‘class std::__cxx11::match_results<__gnu_cxx::__normal_iterator<const char*, std::__cxx11::basic_string<char> > >’} has no member named ‘first’
   37 |         m.first = 0;
      |           ^~~~~
py-cpp/HumanEval_121_solution.cpp:38:11: error: ‘std::__cxx11::smatch’ {aka ‘class std::__cxx11::match_results<__gnu_cxx::__normal_iterator<const char*, std::__cxx11::basic_string<char> > >’} has no member named ‘second’
   38 |         m.second = 0;
      |           ^~~~~~
py-cpp/HumanEval_121_solution.cpp:39:11: error: ‘std::__cxx11::smatch’ {aka ‘class std::__cxx11::match_results<__gnu_cxx::__normal_iterator<const char*, std::__cxx11::basic_string<char> > >’} has no member named ‘first’
   39 |         m.first = std::stoi(m.second);
      |           ^~~~~
py-cpp/HumanEval_121_solution.cpp:39:31: error: ‘std::__cxx11::smatch’ {aka ‘class std::__cxx11::match_results<__gnu_cxx::__normal_iterator<const char*, std::__cxx11::basic_string<char> > >’} has no member named ‘second’
   39 |         m.first = std::stoi(m.second);
      |                               ^~~~~~
py-cpp/HumanEval_121_solution.cpp:40:11: error: ‘std::__cxx11::smatch’ {aka ‘class std::__cxx11::match_results<__gnu_cxx::__normal_iterator<const char*, std::__cxx11::basic_string<char> > >’} has no member named ‘second’
   40 |         m.second = std::to_string(m.first);
      |           ^~~~~~
py-cpp/HumanEval_121_solution.cpp:40:37: error: ‘std::__cxx11::smatch’ {aka ‘class std::__cxx11::match_results<__gnu_cxx::__normal_iterator<const char*, std::__cxx11::basic_string<char> > >’} has no member named ‘first’
   40 |         m.second = std::to_string(m.first);
      |                                     ^~~~~
py-cpp/HumanEval_121_solution.cpp:41:11: error: ‘std::__cxx11::smatch’ {aka ‘class std::__cxx11::match_results<__gnu_cxx::__normal_iterator<const char*, std::__cxx11::basic_string<char> > >’} has no member named ‘first’
   41 |         m.first = 0;
      |           ^~~~~
py-cpp/HumanEval_121_solution.cpp:42:11: error: ‘std::__cxx11::smatch’ {aka ‘class std::__cxx11::match_results<__gnu_cxx::__normal_iterator<const char*, std::__cxx11::basic_string<char> > >’} has no member named ‘second’
   42 |         m.second = 0;
      |           ^~~~~~
py-cpp/HumanEval_121_solution.cpp:43:11: error: ‘std::__cxx11::smatch’ {aka ‘class std::__cxx11::match_results<__gnu_cxx::__normal_iterator<const char*, std::__cxx11::basic_string<char> > >’} has no member named ‘first’
   43 |         m.first = std::stoi(m.second);
      |           ^~~~~
py-cpp/HumanEval_121_solution.cpp:43:31: error: ‘std::__cxx11::smatch’ {aka ‘class std::__cxx11::match_results<__gnu_cxx::__normal_iterator<const char*, std::__cxx11::basic_string<char> > >’} has no member named ‘second’
   43 |         m.first = std::stoi(m.second);
      |                               ^~~~~~
py-cpp/HumanEval_121_solution.cpp:44:11: error: ‘std::__cxx11::smatch’ {aka ‘class std::__cxx11::match_results<__gnu_cxx::__normal_iterator<const char*, std::__cxx11::basic_string<char> > >’} has no member named ‘second’
   44 |         m.second = std::to_string(m.first);
      |           ^~~~~~
py-cpp/HumanEval_121_solution.cpp:44:37: error: ‘std::__cxx11::smatch’ {aka ‘class std::__cxx11::match_results<__gnu_cxx::__normal_iterator<const char*, std::__cxx11::basic_string<char> > >’} has no member named ‘first’
   44 |         m.second = std::to_string(m.first);
      |                                     ^~~~~
py-cpp/HumanEval_121_solution.cpp:45:11: error: ‘std::__cxx11::smatch’ {aka ‘class std::__cxx11::match_results<__gnu_cxx::__normal_iterator<const char*, std::__cxx11::basic_string<char> > >’} has no member named ‘first’
   45 |         m.first = 0;
      |           ^~~~~
py-cpp/HumanEval_121_solution.cpp:46:11: error: ‘std::__cxx11::smatch’ {aka ‘class std::__cxx11::match_results<__gnu_cxx::__normal_iterator<const char*, std::__cxx11::basic_string<char> > >’} has no member named ‘second’
   46 |         m.second = 0;
      |           ^~~~~~
py-cpp/HumanEval_121_solution.cpp:47:11: error: ‘std::__cxx11::smatch’ {aka ‘class std::__cxx11::match_results<__gnu_cxx::__normal_iterator<const char*, std::__cxx11::basic_string<char> > >’} has no member named ‘first’
   47 |         m.first = std::stoi(m.second);
      |           ^~~~~
py-cpp/HumanEval_121_solution.cpp:47:31: error: ‘std::__cxx11::smatch’ {aka ‘class std::__cxx11::match_results<__gnu_cxx::__normal_iterator<const char*, std::__cxx11::basic_string<char> > >’} has no member named ‘second’
   47 |         m.first = std::stoi(m.second);
      |                               ^~~~~~
py-cpp/HumanEval_121_solution.cpp:48:11: error: ‘std::__cxx11::smatch’ {aka ‘class std::__cxx11::match_results<__gnu_cxx::__normal_iterator<const char*, std::__cxx11::basic_string<char> > >’} has no member named ‘second’
   48 |         m.second = std::to_string(m.first);
      |           ^~~~~~
py-cpp/HumanEval_121_solution.cpp:48:37: error: ‘std::__cxx11::smatch’ {aka ‘class std::__cxx11::match_results<__gnu_cxx::__normal_iterator<const char*, std::__cxx11::basic_string<char> > >’} has no member named ‘first’
   48 |         m.second = std::to_string(m.first);
      |                                     ^~~~~
py-cpp/HumanEval_121_solution.cpp:49:11: error: ‘std::__cxx11::smatch’ {aka ‘class std::__cxx11::match_results<__gnu_cxx::__normal_iterator<const char*, std::__cxx11::basic_string<char> > >’} has no member named ‘first’
   49 |         m.first = 0;
      |           ^~~~~
py-cpp/HumanEval_121_solution.cpp:50:11: error: ‘std::__cxx11::smatch’ {aka ‘class std::__cxx11::match_results<__gnu_cxx::__normal_iterator<const char*, std::__cxx11::basic_string<char> > >’} has no member named ‘second’
   50 |         m.second = 0;
      |           ^~~~~~
py-cpp/HumanEval_121_solution.cpp:51:11: error: ‘std::__cxx11::smatch’ {aka ‘class std::__cxx11::match_results<__gnu_cxx::__normal_iterator<const char*, std::__cxx11::basic_string<char> > >’} has no member named ‘first’
   51 |         m.first = std::stoi(m.second);
      |           ^~~~~
py-cpp/HumanEval_121_solution.cpp:51:31: error: ‘std::__cxx11::smatch’ {aka ‘class std::__cxx11::match_results<__gnu_cxx::__normal_iterator<const char*, std::__cxx11::basic_string<char> > >’} has no member named ‘second’
   51 |         m.first = std::stoi(m.second);
      |                               ^~~~~~
py-cpp/HumanEval_121_solution.cpp:52:11: error: ‘std::__cxx11::smatch’ {aka ‘class std::__cxx11::match_results<__gnu_cxx::__normal_iterator<const char*, std::__cxx11::basic_string<char> > >’} has no member named ‘second’
   52 |         m.second = std::to_string(m.first);
      |           ^~~~~~
py-cpp/HumanEval_121_solution.cpp:52:37: error: ‘std::__cxx11::smatch’ {aka ‘class std::__cxx11::match_results<__gnu_cxx::__normal_iterator<const char*, std::__cxx11::basic_string<char> > >’} has no member named ‘first’
   52 |         m.second = std::to_string(m.first);
      |                                     ^~~~~
py-cpp/HumanEval_121_solution.cpp:53:11: error: ‘std::__cxx11::smatch’ {aka ‘class std::__cxx11::match_results<__gnu_cxx::__normal_iterator<const char*, std::__cxx11::basic_string<char> > >’} has no member named ‘first’
   53 |         m.first = 0;
      |           ^~~~~
py-cpp/HumanEval_121_solution.cpp:54:11: error: ‘std::__cxx11::smatch’ {aka ‘class std::__cxx11::match_results<__gnu_cxx::__normal_iterator<const char*, std::__cxx11::basic_string<char> > >’} has no member named ‘second’
   54 |         m.second = 0;
      |           ^~~~~~
py-cpp/HumanEval_121_solution.cpp:55:11: error: ‘std::__cxx11::smatch’ {aka ‘class std::__cxx11::match_results<__gnu_cxx::__normal_iterator<const char*, std::__cxx11::basic_string<char> > >’} has no member named ‘first’
   55 |         m.first = std::stoi(m.second);
      |           ^~~~~
py-cpp/HumanEval_121_solution.cpp:55:31: error: ‘std::__cxx11::smatch’ {aka ‘class std::__cxx11::match_results<__gnu_cxx::__normal_iterator<const char*, std::__cxx11::basic_string<char> > >’} has no member named ‘second’
   55 |         m.first = std::stoi(m.second);
      |                               ^~~~~~
py-cpp/HumanEval_121_solution.cpp:56:11: error: ‘std::__cxx11::smatch’ {aka ‘class std::__cxx11::match_results<__gnu_cxx::__normal_iterator<const char*, std::__cxx11::basic_string<char> > >’} has no member named ‘second’
   56 |         m.second = std::to_string(m.first);
      |           ^~~~~~
py-cpp/HumanEval_121_solution.cpp:56:37: error: ‘std::__cxx11::smatch’ {aka ‘class std::__cxx11::match_results<__gnu_cxx::__normal_iterator<const char*, std::__cxx11::basic_string<char> > >’} has no member named ‘first’
   56 |         m.second = std::to_string(m.first);
      |                                     ^~~~~
py-cpp/HumanEval_121_solution.cpp:57:11: error: ‘std::__cxx11::smatch’ {aka ‘class std::__cxx11::match_results<__gnu_cxx::__normal_iterator<const char*, std::__cxx11::basic_string<char> > >’} has no member named ‘first’
   57 |         m.first = 0;
      |           ^~~~~
py-cpp/HumanEval_121_solution.cpp:58:11: error: ‘std::__cxx11::smatch’ {aka ‘class std::__cxx11::match_results<__gnu_cxx::__normal_iterator<const char*, std::__cxx11::basic_string<char> > >’} has no member named ‘second’
   58 |         m.second = 0;
      |           ^~~~~~
py-cpp/HumanEval_121_solution.cpp:59:11: error: ‘std::__cxx11::smatch’ {aka ‘class std::__cxx11::match_results<__gnu_cxx::__normal_iterator<const char*, std::__cxx11::basic_string<char> > >’} has no member named ‘first’
   59 |         m.first = std::stoi(m.second);
      |           ^~~~~
py-cpp/HumanEval_121_solution.cpp:59:31: error: ‘std::__cxx11::smatch’ {aka ‘class std::__cxx11::match_results<__gnu_cxx::__normal_iterator<const char*, std::__cxx11::basic_string<char> > >’} has no member named ‘second’
   59 |         m.first = std::stoi(m.second);
      |                               ^~~~~~
py-cpp/HumanEval_121_solution.cpp:60:11: error: ‘std::__cxx11::smatch’ {aka ‘class std::__cxx11::match_results<__gnu_cxx::__normal_iterator<const char*, std::__cxx11::basic_string<char> > >’} has no member named ‘second’
   60 |         m.second = std::to_string(m.first);
      |           ^~~~~~
py-cpp/HumanEval_121_solution.cpp:60:37: error: ‘std::__cxx11::smatch’ {aka ‘class std::__cxx11::match_results<__gnu_cxx::__normal_iterator<const char*, std::__cxx11::basic_string<char> > >’} has no member named ‘first’
   60 |         m.second = std::to_string(m.first);
      |                                     ^~~~~
py-cpp/HumanEval_121_solution.cpp:61:11: error: ‘std::__cxx11::smatch’ {aka ‘class std::__cxx11::match_results<__gnu_cxx::__normal_iterator<const char*, std::__cxx11::basic_string<char> > >’} has no member named ‘first’
   61 |         m.first = 0;
      |           ^~~~~
py-cpp/HumanEval_121_solution.cpp:62:11: error: ‘std::__cxx11::smatch’ {aka ‘class std::__cxx11::match_results<__gnu_cxx::__normal_iterator<const char*, std::__cxx11::basic_string<char> > >’} has no member named ‘second’
   62 |         m.second = 0;
      |           ^~~~~~
py-cpp/HumanEval_121_solution.cpp:63:11: error: ‘std::__cxx11::smatch’ {aka ‘class std::__cxx11::match_results<__gnu_cxx::__normal_iterator<const char*, std::__cxx11::basic_string<char> > >’} has no member named ‘first’
   63 |         m.first = std::stoi(m.second);
      |           ^~~~~
py-cpp/HumanEval_121_solution.cpp:63:31: error: ‘std::__cxx11::smatch’ {aka ‘class std::__cxx11::match_results<__gnu_cxx::__normal_iterator<const char*, std::__cxx11::basic_string<char> > >’} has no member named ‘second’
   63 |         m.first = std::stoi(m.second);
      |                               ^~~~~~
py-cpp/HumanEval_121_solution.cpp:64:11: error: ‘std::__cxx11::smatch’ {aka ‘class std::__cxx11::match_results<__gnu_cxx::__normal_iterator<const char*, std::__cxx11::basic_string<char> > >’} has no member named ‘second’
   64 |         m.second = std::to_string(m.first);
      |           ^~~~~~
py-cpp/HumanEval_121_solution.cpp:64:37: error: ‘std::__cxx11::smatch’ {aka ‘class std::__cxx11::match_results<__gnu_cxx::__normal_iterator<const char*, std::__cxx11::basic_string<char> > >’} has no member named ‘first’
   64 |         m.second = std::to_string(m.first);
      |                                     ^~~~~
py-cpp/HumanEval_121_solution.cpp:65:11: error: ‘std::__cxx11::smatch’ {aka ‘class std::__cxx11::match_results<__gnu_cxx::__normal_iterator<const char*, std::__cxx11::basic_string<char> > >’} has no member named ‘first’
   65 |         m.first = 0;
      |           ^~~~~
py-cpp/HumanEval_121_solution.cpp:66:11: error: ‘std::__cxx11::smatch’ {aka ‘class std::__cxx11::match_results<__gnu_cxx::__normal_iterator<const char*, std::__cxx11::basic_string<char> > >’} has no member named ‘second’
   66 |         m.second = 0;
      |           ^~~~~~
py-cpp/HumanEval_121_solution.cpp:67:11: error: ‘std::__cxx11::smatch’ {aka ‘class std::__cxx11::match_results<__gnu_cxx::__normal_iterator<const char*, std::__cxx11::basic_string<char> > >’} has no member named ‘first’
   67 |         m.first = std::stoi(m.second);
      |           ^~~~~
py-cpp/HumanEval_121_solution.cpp:67:31: error: ‘std::__cxx11::smatch’ {aka ‘class std::__cxx11::match_results<__gnu_cxx::__normal_iterator<const char*, std::__cxx11::basic_string<char> > >’} has no member named ‘second’
   67 |         m.first = std::stoi(m.second);
      |                               ^~~~~~
py-cpp/HumanEval_121_solution.cpp:68:11: error: ‘std::__cxx11::smatch’ {aka ‘class std::__cxx11::match_results<__gnu_cxx::__normal_iterator<const char*, std::__cxx11::basic_string<char> > >’} has no member named ‘second’
   68 |         m.second = std::to_string(m.first);
      |           ^~~~~~
py-cpp/HumanEval_121_solution.cpp:68:37: error: ‘std::__cxx11::smatch’ {aka ‘class std::__cxx11::match_results<__gnu_cxx::__normal_iterator<const char*, std::__cxx11::basic_string<char> > >’} has no member named ‘first’
   68 |         m.second = std::to_string(m.first);
      |                                     ^~~~~
py-cpp/HumanEval_121_solution.cpp:69:11: error: ‘std::__cxx11::smatch’ {aka ‘class std::__cxx11::match_results<__gnu_cxx::__normal_iterator<const char*, std::__cxx11::basic_string<char> > >’} has no member named ‘first’
   69 |         m.first = 0;
      |           ^~~~~
py-cpp/HumanEval_121_solution.cpp:70:11: error: ‘std::__cxx11::smatch’ {aka ‘class std::__cxx11::match_results<__gnu_cxx::__normal_iterator<const char*, std::__cxx11::basic_string<char> > >’} has no member named ‘second’
   70 |         m.second = 0;
      |           ^~~~~~
py-cpp/HumanEval_121_solution.cpp:71:11: error: ‘std::__cxx11::smatch’ {aka ‘class std::__cxx11::match_results<__gnu_cxx::__normal_iterator<const char*, std::__cxx11::basic_string<char> > >’} has no member named ‘first’
   71 |         m.first = std::stoi(m.second);
      |           ^~~~~
py-cpp/HumanEval_121_solution.cpp:71:31: error: ‘std::__cxx11::smatch’ {aka ‘class std::__cxx11::match_results<__gnu_cxx::__normal_iterator<const char*, std::__cxx11::basic_string<char> > >’} has no member named ‘second’
   71 |         m.first = std::stoi(m.second);
      |                               ^~~~~~
py-cpp/HumanEval_121_solution.cpp:72:11: error: ‘std::__cxx11::smatch’ {aka ‘class std::__cxx11::match_results<__gnu_cxx::__normal_iterator<const char*, std::__cxx11::basic_string<char> > >’} has no member named ‘second’
   72 |         m.second = std::to_string(m.first);
      |           ^~~~~~
py-cpp/HumanEval_121_solution.cpp:72:37: error: ‘std::__cxx11::smatch’ {aka ‘class std::__cxx11::match_results<__gnu_cxx::__normal_iterator<const char*, std::__cxx11::basic_string<char> > >’} has no member named ‘first’
   72 |         m.second = std::to_string(m.first);
      |                                     ^~~~~
py-cpp/HumanEval_121_solution.cpp:73:11: error: ‘std::__cxx11::smatch’ {aka ‘class std::__cxx11::match_results<__gnu_cxx::__normal_iterator<const char*, std::__cxx11::basic_string<char> > >’} has no member named ‘first’
   73 |         m.first = 0;
      |           ^~~~~
py-cpp/HumanEval_121_solution.cpp:74:11: error: ‘std::__cxx11::smatch’ {aka ‘class std::__cxx11::match_results<__gnu_cxx::__normal_iterator<const char*, std::__cxx11::basic_string<char> > >’} has no member named ‘second’
   74 |         m.second = 0;
      |           ^~~~~~
py-cpp/HumanEval_121_solution.cpp:75:11: error: ‘std::__cxx11::smatch’ {aka ‘class std::__cxx11::match_results<__gnu_cxx::__normal_iterator<const char*, std::__cxx11::basic_string<char> > >’} has no member named ‘first’
   75 |         m.first = std::stoi(m.second);
      |           ^~~~~
py-cpp/HumanEval_121_solution.cpp:75:31: error: ‘std::__cxx11::smatch’ {aka ‘class std::__cxx11::match_results<__gnu_cxx::__normal_iterator<const char*, std::__cxx11::basic_string<char> > >’} has no member named ‘second’
   75 |         m.first = std::stoi(m.second);
      |                               ^~~~~~
py-cpp/HumanEval_121_solution.cpp:76:11: error: ‘std::__cxx11::smatch’ {aka ‘class std::__cxx11::match_results<__gnu_cxx::__normal_iterator<const char*, std::__cxx11::basic_string<char> > >’} has no member named ‘second’
   76 |         m.second = std::to_string(m.first);
      |           ^~~~~~
py-cpp/HumanEval_121_solution.cpp:76:37: error: ‘std::__cxx11::smatch’ {aka ‘class std::__cxx11::match_results<__gnu_cxx::__normal_iterator<const char*, std::__cxx11::basic_string<char> > >’} has no member named ‘first’
   76 |         m.second = std::to_string(m.first);
      |                                     ^~~~~
py-cpp/HumanEval_121_solution.cpp:77:11: error: ‘std::__cxx11::smatch’ {aka ‘class std::__cxx11::match_results<__gnu_cxx::__normal_iterator<const char*, std::__cxx11::basic_string<char> > >’} has no member named ‘first’
   77 |         m.first = 0;
      |           ^~~~~
py-cpp/HumanEval_121_solution.cpp:78:11: error: ‘std::__cxx11::smatch’ {aka ‘class std::__cxx11::match_results<__gnu_cxx::__normal_iterator<const char*, std::__cxx11::basic_string<char> > >’} has no member named ‘second’
   78 |         m.second = 0;
      |           ^~~~~~
py-cpp/HumanEval_121_solution.cpp:79:11: error: ‘std::__cxx11::smatch’ {aka ‘class std::__cxx11::match_results<__gnu_cxx::__normal_iterator<const char*, std::__cxx11::basic_string<char> > >’} has no member named ‘first’
   79 |         m.first = std::stoi(m.second);
      |           ^~~~~
py-cpp/HumanEval_121_solution.cpp:79:31: error: ‘std::__cxx11::smatch’ {aka ‘class std::__cxx11::match_results<__gnu_cxx::__normal_iterator<const char*, std::__cxx11::basic_string<char> > >’} has no member named ‘second’
   79 |         m.first = std::stoi(m.second);
      |                               ^~~~~~
py-cpp/HumanEval_121_solution.cpp:80:11: error: ‘std::__cxx11::smatch’ {aka ‘class std::__cxx11::match_results<__gnu_cxx::__normal_iterator<const char*, std::__cxx11::basic_string<char> > >’} has no member named ‘second’
   80 |         m.second = std::to_string(m.first);
      |           ^~~~~~
py-cpp/HumanEval_121_solution.cpp:80:37: error: ‘std::__cxx11::smatch’ {aka ‘class std::__cxx11::match_results<__gnu_cxx::__normal_iterator<const char*, std::__cxx11::basic_string<char> > >’} has no member named ‘first’
   80 |         m.second = std::to_string(m.first);
      |                                     ^~~~~
py-cpp/HumanEval_121_solution.cpp:81:11: error: ‘std::__cxx11::smatch’ {aka ‘class std::__cxx11::match_results<__gnu_cxx::__normal_iterator<const char*, std::__cxx11::basic_string<char> > >’} has no member named ‘first’
   81 |         m.first = 0;
      |           ^~~~~
py-cpp/HumanEval_121_solution.cpp:82:11: error: ‘std::__cxx11::smatch’ {aka ‘class std::__cxx11::match_results<__gnu_cxx::__normal_iterator<const char*, std::__cxx11::basic_string<char> > >’} has no member named ‘second’
   82 |         m.second = 0;
      |           ^~~~~~
py-cpp/HumanEval_121_solution.cpp:83:11: error: ‘std::__cxx11::smatch’ {aka ‘class std::__cxx11::match_results<__gnu_cxx::__normal_iterator<const char*, std::__cxx11::basic_string<char> > >’} has no member named ‘first’
   83 |         m.first = std::stoi(m.second);
      |           ^~~~~
py-cpp/HumanEval_121_solution.cpp:83:31: error: ‘std::__cxx11::smatch’ {aka ‘class std::__cxx11::match_results<__gnu_cxx::__normal_iterator<const char*, std::__cxx11::basic_string<char> > >’} has no member named ‘second’
   83 |         m.first = std::stoi(m.second);
      |                               ^~~~~~
py-cpp/HumanEval_121_solution.cpp:84:11: error: ‘std::__cxx11::smatch’ {aka ‘class std::__cxx11::match_results<__gnu_cxx::__normal_iterator<const char*, std::__cxx11::basic_string<char> > >’} has no member named ‘second’
   84 |         m.second = std::to_string(m.first);
      |           ^~~~~~
py-cpp/HumanEval_121_solution.cpp:84:37: error: ‘std::__cxx11::smatch’ {aka ‘class std::__cxx11::match_results<__gnu_cxx::__normal_iterator<const char*, std::__cxx11::basic_string<char> > >’} has no member named ‘first’
   84 |         m.second = std::to_string(m.first);
      |                                     ^~~~~
py-cpp/HumanEval_121_solution.cpp:86:1: error: expected unqualified-id before ‘}’ token
   86 | }
      | ^
py-cpp/HumanEval_121_solution.cpp:87:12: error: a function-definition is not allowed here before ‘{’ token
   87 | int main() {
      |            ^
py-cpp/HumanEval_121_solution.cpp:96:1: error: expected ‘}’ at end of input
   96 | }
      | ^
py-cpp/HumanEval_121_solution.cpp:10:38: note: to match this ‘{’
   10 | long solution(std::vector<long> lst) {
      |                                      ^
py-cpp/HumanEval_121_solution.cpp:96:1: warning: no return statement in function returning non-void [-Wreturn-type]
   96 | }
      | ^

2024-12-10 11:12:37 - ERROR - Failed to compile py-cpp/HumanEval_98_count_upper.cpp. Error:
py-cpp/HumanEval_98_count_upper.cpp: In function ‘long int count_upper(std::string)’:
py-cpp/HumanEval_98_count_upper.cpp:19:18: error: ‘const value_type’ {aka ‘const class std::__cxx11::sub_match<__gnu_cxx::__normal_iterator<const char*, std::__cxx11::basic_string<char> > >’} has no member named ‘size’
   19 |         if (m[i].size() == 1) {
      |                  ^~~~
py-cpp/HumanEval_98_count_upper.cpp:20:38: error: no match for ‘operator[]’ (operand types are ‘const value_type’ {aka ‘const std::__cxx11::sub_match<__gnu_cxx::__normal_iterator<const char*, std::__cxx11::basic_string<char> > >’} and ‘int’)
   20 |             count += count_upper(m[i][0]);
      |                                      ^
py-cpp/HumanEval_98_count_upper.cpp:22:38: error: no match for ‘operator[]’ (operand types are ‘const value_type’ {aka ‘const std::__cxx11::sub_match<__gnu_cxx::__normal_iterator<const char*, std::__cxx11::basic_string<char> > >’} and ‘int’)
   22 |             count += count_upper(m[i][0]);
      |                                      ^
py-cpp/HumanEval_98_count_upper.cpp:23:38: error: no match for ‘operator[]’ (operand types are ‘const value_type’ {aka ‘const std::__cxx11::sub_match<__gnu_cxx::__normal_iterator<const char*, std::__cxx11::basic_string<char> > >’} and ‘int’)
   23 |             count += count_upper(m[i][1]);
      |                                      ^

2024-12-10 11:12:38 - ERROR - Failed to compile py-cpp/HumanEval_153_Strongest_Extension.cpp. Error:
py-cpp/HumanEval_153_Strongest_Extension.cpp: In function ‘std::string Strongest_Extension(std::string, std::vector<std::__cxx11::basic_string<char> >)’:
py-cpp/HumanEval_153_Strongest_Extension.cpp:93:1: error: expected unqualified-id before ‘}’ token
   93 | }
      | ^
py-cpp/HumanEval_153_Strongest_Extension.cpp:93:1: warning: no return statement in function returning non-void [-Wreturn-type]

2024-12-10 11:12:38 - INFO - Successfully compiled: py-cpp/HumanEval_138_is_equal_to_sum_even.cpp
2024-12-10 11:12:39 - ERROR - Failed to compile py-cpp/HumanEval_127_intersection.cpp. Error:
py-cpp/HumanEval_127_intersection.cpp: In function ‘std::string intersection(std::tuple<long int, long int>, std::tuple<long int, long int>)’:
py-cpp/HumanEval_127_intersection.cpp:12:24: error: no match for ‘operator[]’ (operand types are ‘std::tuple<long int, long int>’ and ‘int’)
   12 |     long l1 = interval1[0], r1 = interval1[1], l2 = interval2[0], r2 = interval2[1];
      |                        ^
py-cpp/HumanEval_127_intersection.cpp:13:15: error: ‘r1’ was not declared in this scope; did you mean ‘l1’?
   13 |     if (l1 == r1) {
      |               ^~
      |               l1
py-cpp/HumanEval_127_intersection.cpp:15:16: error: ‘l2’ was not declared in this scope; did you mean ‘l1’?
   15 |     } else if (l2 == r2) {
      |                ^~
      |                l1
py-cpp/HumanEval_127_intersection.cpp:15:22: error: ‘r2’ was not declared in this scope
   15 |     } else if (l2 == r2) {
      |                      ^~

2024-12-10 11:12:39 - INFO - Successfully compiled: py-cpp/HumanEval_10_make_palindrome.cpp
2024-12-10 11:12:40 - INFO - Successfully compiled: py-cpp/HumanEval_88_sort_array.cpp
2024-12-10 11:12:41 - INFO - Successfully compiled: py-cpp/HumanEval_79_decimal_to_binary.cpp
2024-12-10 11:12:41 - ERROR - Failed to compile py-cpp/HumanEval_103_rounded_avg.cpp. Error:
py-cpp/HumanEval_103_rounded_avg.cpp: In function ‘Union_std_string_long rounded_avg(long int, long int)’:
py-cpp/HumanEval_103_rounded_avg.cpp:22:38: error: could not convert ‘<brace-enclosed initializer list>()’ from ‘<brace-enclosed initializer list>’ to ‘Union_std_string_long’
   22 |     union Union_std_string_long u = {};
      |                                      ^
      |                                      |
      |                                      <brace-enclosed initializer list>
py-cpp/HumanEval_103_rounded_avg.cpp:24:28: error: no match for ‘operator=’ (operand types are ‘std::string’ {aka ‘std::__cxx11::basic_string<char>’} and ‘Union_std_string_long’)
   24 |     u.f0 = rounded_avg(n, m);
      |                            ^
In file included from /usr/include/c++/9/string:55,
                 from /usr/include/c++/9/bits/locale_classes.h:40,
                 from /usr/include/c++/9/bits/ios_base.h:41,
                 from /usr/include/c++/9/ios:42,
                 from /usr/include/c++/9/istream:38,
                 from /usr/include/c++/9/sstream:38,
                 from /usr/include/c++/9/complex:45,
                 from /usr/include/c++/9/ccomplex:39,
                 from /usr/include/x86_64-linux-gnu/c++/9/bits/stdc++.h:54,
                 from py-cpp/HumanEval_103_rounded_avg.cpp:2:
/usr/include/c++/9/bits/basic_string.h:669:7: note: candidate: ‘std::__cxx11::basic_string<_CharT, _Traits, _Alloc>& std::__cxx11::basic_string<_CharT, _Traits, _Alloc>::operator=(const std::__cxx11::basic_string<_CharT, _Traits, _Alloc>&) [with _CharT = char; _Traits = std::char_traits<char>; _Alloc = std::allocator<char>]’
  669 |       operator=(const basic_string& __str)
      |       ^~~~~~~~
/usr/include/c++/9/bits/basic_string.h:669:37: note:   no known conversion for argument 1 from ‘Union_std_string_long’ to ‘const std::__cxx11::basic_string<char>&’
  669 |       operator=(const basic_string& __str)
      |                 ~~~~~~~~~~~~~~~~~~~~^~~~~
/usr/include/c++/9/bits/basic_string.h:708:7: note: candidate: ‘std::__cxx11::basic_string<_CharT, _Traits, _Alloc>& std::__cxx11::basic_string<_CharT, _Traits, _Alloc>::operator=(const _CharT*) [with _CharT = char; _Traits = std::char_traits<char>; _Alloc = std::allocator<char>]’
  708 |       operator=(const _CharT* __s)
      |       ^~~~~~~~
/usr/include/c++/9/bits/basic_string.h:708:31: note:   no known conversion for argument 1 from ‘Union_std_string_long’ to ‘const char*’
  708 |       operator=(const _CharT* __s)
      |                 ~~~~~~~~~~~~~~^~~
/usr/include/c++/9/bits/basic_string.h:719:7: note: candidate: ‘std::__cxx11::basic_string<_CharT, _Traits, _Alloc>& std::__cxx11::basic_string<_CharT, _Traits, _Alloc>::operator=(_CharT) [with _CharT = char; _Traits = std::char_traits<char>; _Alloc = std::allocator<char>]’
  719 |       operator=(_CharT __c)
      |       ^~~~~~~~
/usr/include/c++/9/bits/basic_string.h:719:24: note:   no known conversion for argument 1 from ‘Union_std_string_long’ to ‘char’
  719 |       operator=(_CharT __c)
      |                 ~~~~~~~^~~
/usr/include/c++/9/bits/basic_string.h:736:7: note: candidate: ‘std::__cxx11::basic_string<_CharT, _Traits, _Alloc>& std::__cxx11::basic_string<_CharT, _Traits, _Alloc>::operator=(std::__cxx11::basic_string<_CharT, _Traits, _Alloc>&&) [with _CharT = char; _Traits = std::char_traits<char>; _Alloc = std::allocator<char>]’
  736 |       operator=(basic_string&& __str)
      |       ^~~~~~~~
/usr/include/c++/9/bits/basic_string.h:736:32: note:   no known conversion for argument 1 from ‘Union_std_string_long’ to ‘std::__cxx11::basic_string<char>&&’
  736 |       operator=(basic_string&& __str)
      |                 ~~~~~~~~~~~~~~~^~~~~
/usr/include/c++/9/bits/basic_string.h:799:7: note: candidate: ‘std::__cxx11::basic_string<_CharT, _Traits, _Alloc>& std::__cxx11::basic_string<_CharT, _Traits, _Alloc>::operator=(std::initializer_list<_Tp>) [with _CharT = char; _Traits = std::char_traits<char>; _Alloc = std::allocator<char>]’
  799 |       operator=(initializer_list<_CharT> __l)
      |       ^~~~~~~~
/usr/include/c++/9/bits/basic_string.h:799:42: note:   no known conversion for argument 1 from ‘Union_std_string_long’ to ‘std::initializer_list<char>’
  799 |       operator=(initializer_list<_CharT> __l)
      |                 ~~~~~~~~~~~~~~~~~~~~~~~~~^~~
py-cpp/HumanEval_103_rounded_avg.cpp:25:12: error: use of deleted function ‘Union_std_string_long::Union_std_string_long(const Union_std_string_long&)’
   25 |     return u;
      |            ^
py-cpp/HumanEval_103_rounded_avg.cpp:10:7: note: ‘Union_std_string_long::Union_std_string_long(const Union_std_string_long&)’ is implicitly deleted because the default definition would be ill-formed:
   10 | union Union_std_string_long{
      |       ^~~~~~~~~~~~~~~~~~~~~
py-cpp/HumanEval_103_rounded_avg.cpp:11:17: error: union member ‘Union_std_string_long::f0’ with non-trivial ‘std::__cxx11::basic_string<_CharT, _Traits, _Alloc>::basic_string(const std::__cxx11::basic_string<_CharT, _Traits, _Alloc>&) [with _CharT = char; _Traits = std::char_traits<char>; _Alloc = std::allocator<char>]’
   11 |     std::string f0;
      |                 ^~

2024-12-10 11:12:42 - ERROR - Failed to compile py-cpp/HumanEval_96_count_up_to.cpp. Error:
py-cpp/HumanEval_96_count_up_to.cpp: In function ‘std::vector<long int> count_up_to(long int)’:
py-cpp/HumanEval_96_count_up_to.cpp:17:17: error: conflicting declaration ‘std::__cxx11::smatch m’
   17 |     std::smatch m;
      |                 ^
py-cpp/HumanEval_96_count_up_to.cpp:12:36: note: previous declaration as ‘std::unordered_map<long int, long int> m’
   12 |     std::unordered_map<long, long> m;
      |                                    ^
py-cpp/HumanEval_96_count_up_to.cpp:18:30: error: no matching function for call to ‘regex_match(long int&, std::unordered_map<long int, long int>&, std::__cxx11::regex&)’
   18 |     std::regex_match(n, m, rx);
      |                              ^
In file included from /usr/include/c++/9/regex:62,
                 from /usr/include/x86_64-linux-gnu/c++/9/bits/stdc++.h:110,
                 from py-cpp/HumanEval_96_count_up_to.cpp:2:
/usr/include/c++/9/bits/regex.h:2059:5: note: candidate: ‘template<class _Bi_iter, class _Alloc, class _Ch_type, class _Rx_traits> bool std::regex_match(_Bi_iter, _Bi_iter, std::__cxx11::match_results<_BiIter, _Alloc>&, const std::__cxx11::basic_regex<_CharT, _TraitsT>&, std::regex_constants::match_flag_type)’
 2059 |     regex_match(_Bi_iter     __s,
      |     ^~~~~~~~~~~
/usr/include/c++/9/bits/regex.h:2059:5: note:   template argument deduction/substitution failed:
py-cpp/HumanEval_96_count_up_to.cpp:18:30: note:   deduced conflicting types for parameter ‘_Bi_iter’ (‘long int’ and ‘std::unordered_map<long int, long int>’)
   18 |     std::regex_match(n, m, rx);
      |                              ^
In file included from /usr/include/c++/9/regex:62,
                 from /usr/include/x86_64-linux-gnu/c++/9/bits/stdc++.h:110,
                 from py-cpp/HumanEval_96_count_up_to.cpp:2:
/usr/include/c++/9/bits/regex.h:2087:5: note: candidate: ‘template<class _Bi_iter, class _Ch_type, class _Rx_traits> bool std::regex_match(_Bi_iter, _Bi_iter, const std::__cxx11::basic_regex<_Ch_type, _Rx_traits>&, std::regex_constants::match_flag_type)’
 2087 |     regex_match(_Bi_iter __first, _Bi_iter __last,
      |     ^~~~~~~~~~~
/usr/include/c++/9/bits/regex.h:2087:5: note:   template argument deduction/substitution failed:
py-cpp/HumanEval_96_count_up_to.cpp:18:30: note:   deduced conflicting types for parameter ‘_Bi_iter’ (‘long int’ and ‘std::unordered_map<long int, long int>’)
   18 |     std::regex_match(n, m, rx);
      |                              ^
In file included from /usr/include/c++/9/regex:62,
                 from /usr/include/x86_64-linux-gnu/c++/9/bits/stdc++.h:110,
                 from py-cpp/HumanEval_96_count_up_to.cpp:2:
/usr/include/c++/9/bits/regex.h:2112:5: note: candidate: ‘template<class _Ch_type, class _Alloc, class _Rx_traits> bool std::regex_match(const _Ch_type*, std::__cxx11::match_results<const _Ch_type*, _Alloc>&, const std::__cxx11::basic_regex<_Ch_type, _Rx_traits>&, std::regex_constants::match_flag_type)’
 2112 |     regex_match(const _Ch_type* __s,
      |     ^~~~~~~~~~~
/usr/include/c++/9/bits/regex.h:2112:5: note:   template argument deduction/substitution failed:
py-cpp/HumanEval_96_count_up_to.cpp:18:30: note:   mismatched types ‘const _Ch_type*’ and ‘long int’
   18 |     std::regex_match(n, m, rx);
      |                              ^
In file included from /usr/include/c++/9/regex:62,
                 from /usr/include/x86_64-linux-gnu/c++/9/bits/stdc++.h:110,
                 from py-cpp/HumanEval_96_count_up_to.cpp:2:
/usr/include/c++/9/bits/regex.h:2136:5: note: candidate: ‘template<class _Ch_traits, class _Ch_alloc, class _Alloc, class _Ch_type, class _Rx_traits> bool std::regex_match(const std::__cxx11::basic_string<_Ch_type, _Ch_traits, _Ch_alloc>&, std::__cxx11::match_results<typename std::__cxx11::basic_string<_Ch_type, _Ch_traits, _Ch_alloc>::const_iterator, _Alloc>&, const std::__cxx11::basic_regex<_Ch_type, _Rx_traits>&, std::regex_constants::match_flag_type)’
 2136 |     regex_match(const basic_string<_Ch_type, _Ch_traits, _Ch_alloc>& __s,
      |     ^~~~~~~~~~~
/usr/include/c++/9/bits/regex.h:2136:5: note:   template argument deduction/substitution failed:
py-cpp/HumanEval_96_count_up_to.cpp:18:30: note:   mismatched types ‘const std::__cxx11::basic_string<_Ch_type, _Ch_traits, _Ch_alloc>’ and ‘long int’
   18 |     std::regex_match(n, m, rx);
      |                              ^
In file included from /usr/include/c++/9/regex:62,
                 from /usr/include/x86_64-linux-gnu/c++/9/bits/stdc++.h:110,
                 from py-cpp/HumanEval_96_count_up_to.cpp:2:
/usr/include/c++/9/bits/regex.h:2150:5: note: candidate: ‘template<class _Ch_traits, class _Ch_alloc, class _Alloc, class _Ch_type, class _Rx_traits> bool std::regex_match(const std::__cxx11::basic_string<_Ch_type, _Ch_traits, _Ch_alloc>&&, std::__cxx11::match_results<typename std::__cxx11::basic_string<_Ch_type, _Ch_traits, _Ch_alloc>::const_iterator, _Alloc>&, const std::__cxx11::basic_regex<_Ch_type, _Rx_traits>&, std::regex_constants::match_flag_type)’ <deleted>
 2150 |     regex_match(const basic_string<_Ch_type, _Ch_traits, _Ch_alloc>&&,
      |     ^~~~~~~~~~~
/usr/include/c++/9/bits/regex.h:2150:5: note:   template argument deduction/substitution failed:
py-cpp/HumanEval_96_count_up_to.cpp:18:30: note:   mismatched types ‘const std::__cxx11::basic_string<_Ch_type, _Ch_traits, _Ch_alloc>’ and ‘long int’
   18 |     std::regex_match(n, m, rx);
      |                              ^
In file included from /usr/include/c++/9/regex:62,
                 from /usr/include/x86_64-linux-gnu/c++/9/bits/stdc++.h:110,
                 from py-cpp/HumanEval_96_count_up_to.cpp:2:
/usr/include/c++/9/bits/regex.h:2172:5: note: candidate: ‘template<class _Ch_type, class _Rx_traits> bool std::regex_match(const _Ch_type*, const std::__cxx11::basic_regex<_Ch_type, _Rx_traits>&, std::regex_constants::match_flag_type)’
 2172 |     regex_match(const _Ch_type* __s,
      |     ^~~~~~~~~~~
/usr/include/c++/9/bits/regex.h:2172:5: note:   template argument deduction/substitution failed:
py-cpp/HumanEval_96_count_up_to.cpp:18:30: note:   mismatched types ‘const _Ch_type*’ and ‘long int’
   18 |     std::regex_match(n, m, rx);
      |                              ^
In file included from /usr/include/c++/9/regex:62,
                 from /usr/include/x86_64-linux-gnu/c++/9/bits/stdc++.h:110,
                 from py-cpp/HumanEval_96_count_up_to.cpp:2:
/usr/include/c++/9/bits/regex.h:2194:5: note: candidate: ‘template<class _Ch_traits, class _Str_allocator, class _Ch_type, class _Rx_traits> bool std::regex_match(const std::__cxx11::basic_string<_Ch_type, _Ch_traits, _Str_allocator>&, const std::__cxx11::basic_regex<_CharT, _TraitsT>&, std::regex_constants::match_flag_type)’
 2194 |     regex_match(const basic_string<_Ch_type, _Ch_traits, _Str_allocator>& __s,
      |     ^~~~~~~~~~~
/usr/include/c++/9/bits/regex.h:2194:5: note:   template argument deduction/substitution failed:
py-cpp/HumanEval_96_count_up_to.cpp:18:30: note:   mismatched types ‘const std::__cxx11::basic_string<_Ch_type, _Ch_traits, _Str_allocator>’ and ‘long int’
   18 |     std::regex_match(n, m, rx);
      |                              ^
py-cpp/HumanEval_96_count_up_to.cpp:20:28: error: request for member ‘first’ in ‘m.std::unordered_map<long int, long int>::operator[](((std::unordered_map<long int, long int>::key_type)i))’, which is of non-class type ‘std::unordered_map<long int, long int>::mapped_type’ {aka ‘long int’}
   20 |         res.push_back(m[i].first);
      |                            ^~~~~

2024-12-10 11:12:43 - INFO - Successfully compiled: py-cpp/HumanEval_87_get_row.cpp
2024-12-10 11:12:44 - INFO - Successfully compiled: py-cpp/HumanEval_2_truncate_number.cpp
2024-12-10 11:12:44 - INFO - Successfully compiled: py-cpp/HumanEval_41_car_race_collision.cpp
2024-12-10 11:12:45 - INFO - Successfully compiled: py-cpp/HumanEval_123_get_odd_collatz.cpp
2024-12-10 11:12:46 - INFO - Successfully compiled: py-cpp/HumanEval_107_even_odd_palindrome.cpp
2024-12-10 11:12:46 - ERROR - Failed to compile py-cpp/HumanEval_0_has_close_elements.cpp. Error:
py-cpp/HumanEval_0_has_close_elements.cpp: In function ‘bool has_close_elements(std::vector<float>, float)’:
py-cpp/HumanEval_0_has_close_elements.cpp:14:32: error: could not convert ‘it.__gnu_cxx::__normal_iterator<float*, std::vector<float> >::operator*()’ from ‘float’ to ‘std::vector<float>’
   14 |         if (has_close_elements(*it, threshold)) {
      |                                ^~~
      |                                |
      |                                float

2024-12-10 11:12:47 - ERROR - Failed to compile py-cpp/HumanEval_3_below_zero.cpp. Error:
py-cpp/HumanEval_3_below_zero.cpp: In function ‘bool below_zero(std::vector<long int>)’:
py-cpp/HumanEval_3_below_zero.cpp:14:24: error: could not convert ‘it.__gnu_cxx::__normal_iterator<long int*, std::vector<long int> >::operator*()’ from ‘long int’ to ‘std::vector<long int>’
   14 |         if (below_zero(*it)) {
      |                        ^~~
      |                        |
      |                        long int

2024-12-10 11:12:47 - ERROR - Failed to compile py-cpp/HumanEval_9_rolling_max.cpp. Error:
py-cpp/HumanEval_9_rolling_max.cpp: In function ‘std::vector<long int> rolling_max(std::vector<long int>)’:
py-cpp/HumanEval_9_rolling_max.cpp:14:15: error: ‘max’ was not declared in this scope; did you mean ‘std::max’?
   14 |         *it = max(n, *it);
      |               ^~~
      |               std::max
In file included from /usr/include/c++/9/algorithm:62,
                 from /usr/include/x86_64-linux-gnu/c++/9/bits/stdc++.h:65,
                 from py-cpp/HumanEval_9_rolling_max.cpp:2:
/usr/include/c++/9/bits/stl_algo.h:3468:5: note: ‘std::max’ declared here
 3468 |     max(initializer_list<_Tp> __l, _Compare __comp)
      |     ^~~

2024-12-10 11:12:48 - INFO - Successfully compiled: py-cpp/HumanEval_7_filter_by_substring.cpp
2024-12-10 11:12:49 - INFO - Successfully compiled: py-cpp/HumanEval_101_words_string.cpp
2024-12-10 11:12:50 - ERROR - Failed to compile py-cpp/HumanEval_54_same_chars.cpp. Error:
py-cpp/HumanEval_54_same_chars.cpp: In function ‘bool same_chars(std::string, std::string)’:
py-cpp/HumanEval_54_same_chars.cpp:24:18: error: ‘std::__cxx11::regex’ {aka ‘class std::__cxx11::basic_regex<char>’} has no member named ‘match’
   24 |     if (s0_regex.match(s0_str, sm)) {
      |                  ^~~~~
py-cpp/HumanEval_54_same_chars.cpp:29:18: error: ‘std::__cxx11::regex’ {aka ‘class std::__cxx11::basic_regex<char>’} has no member named ‘match’
   29 |     if (s1_regex.match(s1_str, sm)) {
      |                  ^~~~~
py-cpp/HumanEval_54_same_chars.cpp:49:33: error: expected ‘)’ before ‘}’ token
   49 |            same_chars(s0_str, s1
      |                      ~          ^
      |                                 )
   50 | }
      | ~                                
py-cpp/HumanEval_54_same_chars.cpp:49:33: error: expected ‘;’ before ‘}’ token
   49 |            same_chars(s0_str, s1
      |                                 ^
      |                                 ;
   50 | }
      | ~                                

2024-12-10 11:12:50 - INFO - Successfully compiled: py-cpp/HumanEval_115_max_fill.cpp
2024-12-10 11:12:51 - INFO - Successfully compiled: py-cpp/HumanEval_55_fib.cpp
2024-12-10 11:12:52 - INFO - Successfully compiled: py-cpp/HumanEval_147_get_max_triples.cpp
2024-12-10 11:12:52 - ERROR - Failed to compile py-cpp/HumanEval_137_compare_one.cpp. Error:
py-cpp/HumanEval_137_compare_one.cpp:29:10: error: ‘nullopt’ in namespace ‘std’ does not name a type
   29 |     std::nullopt f3;    Union_long_float_std_string_std_nullopt(long _f0) : f0(_f0) {}
      |          ^~~~~~~
py-cpp/HumanEval_137_compare_one.cpp:32:57: error: expected ‘)’ before ‘_f3’
   32 |     Union_long_float_std_string_std_nullopt(std::nullopt _f3) : f3(_f3) {}
      |                                            ~            ^~~~
      |                                                         )
py-cpp/HumanEval_137_compare_one.cpp:40:31: error: ‘std::nullopt’ has not been declared
   40 |     }    bool operator==(std::nullopt f) {
      |                               ^~~~~~~
py-cpp/HumanEval_137_compare_one.cpp: In member function ‘bool Union_long_float_std_string_std_nullopt::operator==(int)’:
py-cpp/HumanEval_137_compare_one.cpp:41:16: error: ‘f3’ was not declared in this scope; did you mean ‘f’?
   41 |         return f3 == f ;
      |                ^~
      |                f
py-cpp/HumanEval_137_compare_one.cpp: In function ‘Union_long_float_std_string_std_nullopt compare_one(Union_long_float_std_string, Union_long_float_std_string)’:
py-cpp/HumanEval_137_compare_one.cpp:48:22: error: ‘union Union_long_float_std_string’ has no member named ‘f3’; did you mean ‘f0’?
   48 |                 if(a.f3 == b.f3) {
      |                      ^~
      |                      f0
py-cpp/HumanEval_137_compare_one.cpp:48:30: error: ‘union Union_long_float_std_string’ has no member named ‘f3’; did you mean ‘f0’?
   48 |                 if(a.f3 == b.f3) {
      |                              ^~
      |                              f0
py-cpp/HumanEval_137_compare_one.cpp:49:28: error: could not convert ‘a’ from ‘Union_long_float_std_string’ to ‘Union_long_float_std_string_std_nullopt’
   49 |                     return a;
      |                            ^
      |                            |
      |                            Union_long_float_std_string
py-cpp/HumanEval_137_compare_one.cpp:54:12: error: could not convert ‘a’ from ‘Union_long_float_std_string’ to ‘Union_long_float_std_string_std_nullopt’
   54 |     return a;
      |            ^
      |            |
      |            Union_long_float_std_string
In file included from /usr/include/c++/9/cassert:44,
                 from /usr/include/x86_64-linux-gnu/c++/9/bits/stdc++.h:33,
                 from py-cpp/HumanEval_137_compare_one.cpp:2:
py-cpp/HumanEval_137_compare_one.cpp: In function ‘int main()’:
py-cpp/HumanEval_137_compare_one.cpp:58:26: error: conversion from ‘int’ to ‘Union_long_float_std_string’ is ambiguous
   58 |     assert(candidate(1, 2) == 2);
      |                          ^
py-cpp/HumanEval_137_compare_one.cpp:14:5: note: candidate: ‘Union_long_float_std_string::Union_long_float_std_string(float)’
   14 |     Union_long_float_std_string(float _f1) : f1(_f1) {}
      |     ^~~~~~~~~~~~~~~~~~~~~~~~~~~
py-cpp/HumanEval_137_compare_one.cpp:13:24: note: candidate: ‘Union_long_float_std_string::Union_long_float_std_string(long int)’
   13 |     std::string f2;    Union_long_float_std_string(long _f0) : f0(_f0) {}
      |                        ^~~~~~~~~~~~~~~~~~~~~~~~~~~
In file included from /usr/include/c++/9/cassert:44,
                 from /usr/include/x86_64-linux-gnu/c++/9/bits/stdc++.h:33,
                 from py-cpp/HumanEval_137_compare_one.cpp:2:
py-cpp/HumanEval_137_compare_one.cpp:59:29: error: conversion from ‘int’ to ‘Union_long_float_std_string’ is ambiguous
   59 |     assert(candidate(1, 2.5f) == 2.5f);
      |                             ^
py-cpp/HumanEval_137_compare_one.cpp:14:5: note: candidate: ‘Union_long_float_std_string::Union_long_float_std_string(float)’
   14 |     Union_long_float_std_string(float _f1) : f1(_f1) {}
      |     ^~~~~~~~~~~~~~~~~~~~~~~~~~~
py-cpp/HumanEval_137_compare_one.cpp:13:24: note: candidate: ‘Union_long_float_std_string::Union_long_float_std_string(long int)’
   13 |     std::string f2;    Union_long_float_std_string(long _f0) : f0(_f0) {}
      |                        ^~~~~~~~~~~~~~~~~~~~~~~~~~~
In file included from /usr/include/c++/9/cassert:44,
                 from /usr/include/x86_64-linux-gnu/c++/9/bits/stdc++.h:33,
                 from py-cpp/HumanEval_137_compare_one.cpp:2:
py-cpp/HumanEval_137_compare_one.cpp:60:26: error: conversion from ‘int’ to ‘Union_long_float_std_string’ is ambiguous
   60 |     assert(candidate(2, 3) == 3);
      |                          ^
py-cpp/HumanEval_137_compare_one.cpp:14:5: note: candidate: ‘Union_long_float_std_string::Union_long_float_std_string(float)’
   14 |     Union_long_float_std_string(float _f1) : f1(_f1) {}
      |     ^~~~~~~~~~~~~~~~~~~~~~~~~~~
py-cpp/HumanEval_137_compare_one.cpp:13:24: note: candidate: ‘Union_long_float_std_string::Union_long_float_std_string(long int)’
   13 |     std::string f2;    Union_long_float_std_string(long _f0) : f0(_f0) {}
      |                        ^~~~~~~~~~~~~~~~~~~~~~~~~~~
In file included from /usr/include/c++/9/cassert:44,
                 from /usr/include/x86_64-linux-gnu/c++/9/bits/stdc++.h:33,
                 from py-cpp/HumanEval_137_compare_one.cpp:2:
py-cpp/HumanEval_137_compare_one.cpp:61:26: error: conversion from ‘int’ to ‘Union_long_float_std_string’ is ambiguous
   61 |     assert(candidate(5, 6) == 6);
      |                          ^
py-cpp/HumanEval_137_compare_one.cpp:14:5: note: candidate: ‘Union_long_float_std_string::Union_long_float_std_string(float)’
   14 |     Union_long_float_std_string(float _f1) : f1(_f1) {}
      |     ^~~~~~~~~~~~~~~~~~~~~~~~~~~
py-cpp/HumanEval_137_compare_one.cpp:13:24: note: candidate: ‘Union_long_float_std_string::Union_long_float_std_string(long int)’
   13 |     std::string f2;    Union_long_float_std_string(long _f0) : f0(_f0) {}
      |                        ^~~~~~~~~~~~~~~~~~~~~~~~~~~
In file included from /usr/include/c++/9/cassert:44,
                 from /usr/include/x86_64-linux-gnu/c++/9/bits/stdc++.h:33,
                 from py-cpp/HumanEval_137_compare_one.cpp:2:
py-cpp/HumanEval_137_compare_one.cpp:62:30: error: conversion from ‘int’ to ‘Union_long_float_std_string’ is ambiguous
   62 |     assert(candidate(1, "2,3") == "2,3");
      |                              ^
py-cpp/HumanEval_137_compare_one.cpp:14:5: note: candidate: ‘Union_long_float_std_string::Union_long_float_std_string(float)’
   14 |     Union_long_float_std_string(float _f1) : f1(_f1) {}
      |     ^~~~~~~~~~~~~~~~~~~~~~~~~~~
py-cpp/HumanEval_137_compare_one.cpp:13:24: note: candidate: ‘Union_long_float_std_string::Union_long_float_std_string(long int)’
   13 |     std::string f2;    Union_long_float_std_string(long _f0) : f0(_f0) {}
      |                        ^~~~~~~~~~~~~~~~~~~~~~~~~~~
In file included from /usr/include/c++/9/cassert:44,
                 from /usr/include/x86_64-linux-gnu/c++/9/bits/stdc++.h:33,
                 from py-cpp/HumanEval_137_compare_one.cpp:2:
py-cpp/HumanEval_137_compare_one.cpp:63:22: error: invalid conversion from ‘const char*’ to ‘long int’ [-fpermissive]
   63 |     assert(candidate("5,1", "6") == "6");
      |                      ^~~~~
      |                      |
      |                      const char*
py-cpp/HumanEval_137_compare_one.cpp:13:57: note:   initializing argument 1 of ‘Union_long_float_std_string::Union_long_float_std_string(long int)’
   13 |     std::string f2;    Union_long_float_std_string(long _f0) : f0(_f0) {}
      |                                                    ~~~~~^~~
In file included from /usr/include/c++/9/cassert:44,
                 from /usr/include/x86_64-linux-gnu/c++/9/bits/stdc++.h:33,
                 from py-cpp/HumanEval_137_compare_one.cpp:2:
py-cpp/HumanEval_137_compare_one.cpp:63:32: error: use of deleted function ‘Union_long_float_std_string::Union_long_float_std_string(const Union_long_float_std_string&)’
   63 |     assert(candidate("5,1", "6") == "6");
      |                                ^
py-cpp/HumanEval_137_compare_one.cpp:10:7: note: ‘Union_long_float_std_string::Union_long_float_std_string(const Union_long_float_std_string&)’ is implicitly deleted because the default definition would be ill-formed:
   10 | union Union_long_float_std_string{
      |       ^~~~~~~~~~~~~~~~~~~~~~~~~~~
py-cpp/HumanEval_137_compare_one.cpp:13:17: error: union member ‘Union_long_float_std_string::f2’ with non-trivial ‘std::__cxx11::basic_string<_CharT, _Traits, _Alloc>::basic_string(const std::__cxx11::basic_string<_CharT, _Traits, _Alloc>&) [with _CharT = char; _Traits = std::char_traits<char>; _Alloc = std::allocator<char>]’
   13 |     std::string f2;    Union_long_float_std_string(long _f0) : f0(_f0) {}
      |                 ^~
py-cpp/HumanEval_137_compare_one.cpp:13:24: note:   after user-defined conversion: ‘Union_long_float_std_string::Union_long_float_std_string(long int)’ <near match>
   13 |     std::string f2;    Union_long_float_std_string(long _f0) : f0(_f0) {}
      |                        ^~~~~~~~~~~~~~~~~~~~~~~~~~~
py-cpp/HumanEval_137_compare_one.cpp:13:24: note:   conversion of argument 1 would be ill-formed:
In file included from /usr/include/c++/9/cassert:44,
                 from /usr/include/x86_64-linux-gnu/c++/9/bits/stdc++.h:33,
                 from py-cpp/HumanEval_137_compare_one.cpp:2:
py-cpp/HumanEval_137_compare_one.cpp:63:22: error: invalid conversion from ‘const char*’ to ‘long int’ [-fpermissive]
   63 |     assert(candidate("5,1", "6") == "6");
      |                      ^~~~~
      |                      |
      |                      const char*
py-cpp/HumanEval_137_compare_one.cpp:64:22: error: invalid conversion from ‘const char*’ to ‘long int’ [-fpermissive]
   64 |     assert(candidate("1", "2") == "2");
      |                      ^~~
      |                      |
      |                      const char*
py-cpp/HumanEval_137_compare_one.cpp:13:57: note:   initializing argument 1 of ‘Union_long_float_std_string::Union_long_float_std_string(long int)’
   13 |     std::string f2;    Union_long_float_std_string(long _f0) : f0(_f0) {}
      |                                                    ~~~~~^~~
In file included from /usr/include/c++/9/cassert:44,
                 from /usr/include/x86_64-linux-gnu/c++/9/bits/stdc++.h:33,
                 from py-cpp/HumanEval_137_compare_one.cpp:2:
py-cpp/HumanEval_137_compare_one.cpp:64:30: error: use of deleted function ‘Union_long_float_std_string::Union_long_float_std_string(const Union_long_float_std_string&)’
   64 |     assert(candidate("1", "2") == "2");
      |                              ^
py-cpp/HumanEval_137_compare_one.cpp:13:24: note:   after user-defined conversion: ‘Union_long_float_std_string::Union_long_float_std_string(long int)’ <near match>
   13 |     std::string f2;    Union_long_float_std_string(long _f0) : f0(_f0) {}
      |                        ^~~~~~~~~~~~~~~~~~~~~~~~~~~
py-cpp/HumanEval_137_compare_one.cpp:13:24: note:   conversion of argument 1 would be ill-formed:
In file included from /usr/include/c++/9/cassert:44,
                 from /usr/include/x86_64-linux-gnu/c++/9/bits/stdc++.h:33,
                 from py-cpp/HumanEval_137_compare_one.cpp:2:
py-cpp/HumanEval_137_compare_one.cpp:64:22: error: invalid conversion from ‘const char*’ to ‘long int’ [-fpermissive]
   64 |     assert(candidate("1", "2") == "2");
      |                      ^~~
      |                      |
      |                      const char*
py-cpp/HumanEval_137_compare_one.cpp:65:22: error: invalid conversion from ‘const char*’ to ‘long int’ [-fpermissive]
   65 |     assert(candidate("1", 1) == std::nullopt);
      |                      ^~~
      |                      |
      |                      const char*
py-cpp/HumanEval_137_compare_one.cpp:13:57: note:   initializing argument 1 of ‘Union_long_float_std_string::Union_long_float_std_string(long int)’
   13 |     std::string f2;    Union_long_float_std_string(long _f0) : f0(_f0) {}
      |                                                    ~~~~~^~~
In file included from /usr/include/c++/9/cassert:44,
                 from /usr/include/x86_64-linux-gnu/c++/9/bits/stdc++.h:33,
                 from py-cpp/HumanEval_137_compare_one.cpp:2:
py-cpp/HumanEval_137_compare_one.cpp:65:28: error: use of deleted function ‘Union_long_float_std_string::Union_long_float_std_string(const Union_long_float_std_string&)’
   65 |     assert(candidate("1", 1) == std::nullopt);
      |                            ^
py-cpp/HumanEval_137_compare_one.cpp:13:24: note:   after user-defined conversion: ‘Union_long_float_std_string::Union_long_float_std_string(long int)’ <near match>
   13 |     std::string f2;    Union_long_float_std_string(long _f0) : f0(_f0) {}
      |                        ^~~~~~~~~~~~~~~~~~~~~~~~~~~
py-cpp/HumanEval_137_compare_one.cpp:13:24: note:   conversion of argument 1 would be ill-formed:
In file included from /usr/include/c++/9/cassert:44,
                 from /usr/include/x86_64-linux-gnu/c++/9/bits/stdc++.h:33,
                 from py-cpp/HumanEval_137_compare_one.cpp:2:
py-cpp/HumanEval_137_compare_one.cpp:65:22: error: invalid conversion from ‘const char*’ to ‘long int’ [-fpermissive]
   65 |     assert(candidate("1", 1) == std::nullopt);
      |                      ^~~
      |                      |
      |                      const char*
py-cpp/HumanEval_137_compare_one.cpp:65:38: error: ‘nullopt’ is not a member of ‘std’
   65 |     assert(candidate("1", 1) == std::nullopt);
      |                                      ^~~~~~~

2024-12-10 11:12:54 - INFO - Successfully compiled: py-cpp/HumanEval_14_all_prefixes.cpp
2024-12-10 11:12:55 - INFO - Successfully compiled: py-cpp/HumanEval_46_fib4.cpp
2024-12-10 11:12:55 - INFO - Successfully compiled: py-cpp/HumanEval_104_unique_digits.cpp
2024-12-10 11:12:56 - INFO - Successfully compiled: py-cpp/HumanEval_76_is_simple_power.cpp
2024-12-10 11:12:57 - ERROR - Failed to compile py-cpp/HumanEval_162_string_to_md5.cpp. Error:
py-cpp/HumanEval_162_string_to_md5.cpp:10:6: error: ‘optional’ in namespace ‘std’ does not name a template type
   10 | std::optional<std::string> string_to_md5(std::string text) {
      |      ^~~~~~~~
py-cpp/HumanEval_162_string_to_md5.cpp:10:1: note: ‘std::optional’ is only available from C++17 onwards
   10 | std::optional<std::string> string_to_md5(std::string text) {
      | ^~~
py-cpp/HumanEval_162_string_to_md5.cpp: In function ‘int main()’:
py-cpp/HumanEval_162_string_to_md5.cpp:62:22: error: ‘string_to_md5’ was not declared in this scope
   62 |     auto candidate = string_to_md5;
      |                      ^~~~~~~~~~~~~
In file included from /usr/include/c++/9/cassert:44,
                 from /usr/include/x86_64-linux-gnu/c++/9/bits/stdc++.h:33,
                 from py-cpp/HumanEval_162_string_to_md5.cpp:2:
py-cpp/HumanEval_162_string_to_md5.cpp:64:36: error: ‘nullopt’ is not a member of ‘std’
   64 |     assert(candidate(("")) == std::nullopt);
      |                                    ^~~~~~~

2024-12-10 11:12:57 - INFO - Successfully compiled: py-cpp/HumanEval_82_prime_length.cpp
2024-12-10 11:12:58 - INFO - Successfully compiled: py-cpp/HumanEval_43_pairs_sum_to_zero.cpp
2024-12-10 11:12:59 - INFO - Successfully compiled: py-cpp/HumanEval_144_simplify.cpp
2024-12-10 11:12:59 - ERROR - Failed to compile py-cpp/HumanEval_31_is_prime.cpp. Error:
py-cpp/HumanEval_31_is_prime.cpp: In function ‘bool is_prime(long int)’:
py-cpp/HumanEval_31_is_prime.cpp:78:35: error: expected ‘;’ before ‘}’ token
   78 |     if (n % 313 == 0) return false
      |                                   ^
      |                                   ;
   79 | }
      | ~                                  
py-cpp/HumanEval_31_is_prime.cpp:79:1: warning: control reaches end of non-void function [-Wreturn-type]
   79 | }
      | ^

2024-12-10 11:13:00 - INFO - Successfully compiled: py-cpp/HumanEval_120_maximum.cpp
2024-12-10 11:13:01 - INFO - Successfully compiled: py-cpp/HumanEval_133_sum_squares.cpp
2024-12-10 11:13:01 - INFO - Successfully compiled: py-cpp/HumanEval_56_correct_bracketing.cpp
2024-12-10 11:13:02 - INFO - Successfully compiled: py-cpp/HumanEval_19_sort_numbers.cpp
2024-12-10 11:13:02 - INFO - Directory: py-cpp
2024-12-10 11:13:02 - INFO - Total Files: 49
2024-12-10 11:13:02 - INFO - Successful Compilations: 32
2024-12-10 11:13:02 - INFO - Failed Compilations: 17
2024-12-10 11:13:02 - INFO - Success Rate: 65.31%
